### PR TITLE
Add Discord link in README

### LINKED ISSUE #17: Can we have a Discord server to collab on this project?
There are a few people who have shown interest in wanting to help out on different parts of the code.

I'd love to see this app flourish and become more widely used in the AC community, so I'd love to help this grow.

### PR DESCRIPTION
**Pull request by Lysoun**
I have added a link to the Discord server in the README. Since the Discord is used for contributors to discuss, I do not think it would be wise to put it on the site so I just added its link in the README.
Closes #17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # ac-nh-turnip-prices
+<a href="https://discord.gg/bRh74X8">
+    <img src="https://img.shields.io/badge/discord-join-7289DA.svg?logo=discord&longCache=true&style=flat" />
+  </a>
```
Comment (theRTC204) [inline README.md:4]: While HTML might work in README files on Github, the correct syntax for markdown files would be

```md
[![discord](https://img.shields.io/badge/discord-join-7289DA.svg?logo=discord&longCache=true&style=flat)](https://discord.gg/bRh74X8)
```

Comment (Lysoun) [reply README.md]: Ah sure, there, done in b83d81c

**Comment by Lysoun** [comment]
No problem, I have seen it on other repositories and thought it was cool so I'm glad I could use it here 😀

**Comment by mikebryant** [comment]
Awesome, thanks :)

**Pull request merged**